products/chef-infra-client.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Chef Infra Client
+category: app
+tags: ruby-runtime
+iconSlug: chef
+permalink: /chef-infra-client
+alternate_urls:
+-   /chef-client
+versionCommand: chef-client -v
+releasePolicyLink: https://docs.chef.io/versions/
+changelogTemplate: "https://docs.chef.io/release_notes_client/__LATEST__"
```

Comment:
Just checked the https://deploy-preview-4388--endoflife-date.netlify.app/chef-infra-client and noticed that the links to latest have 404 errors:
https://docs.chef.io/release_notes_client/18.4.12
https://docs.chef.io/release_notes_client/17.10.95
etc

```suggestion
changelogTemplate: "https://docs.chef.io/release_notes_client/#__LATEST__"
```